### PR TITLE
fix(UserSearch): Fixed search user functionality

### PR DESCRIPTION
**Pull request by JaideepPalit**
> Fixed search user functionality
> * Which issue is this pull request belonging to and how is it solving it? (*#991*)
> * Did you add or update any new dependencies that are required for your change? - No

### Checklist
Must:
- [X] All related issues are referenced in commit messages and in PR

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>

### CODE REVIEW

**Comment by mcjaeger** [comment]
work on a staging instance for us.

**Review by smrutis1 — CHANGES_REQUESTED** [review]

File: libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/couchdb/lucene/LuceneAwareDatabaseConnector.java [hunk 1]
```diff
@@ -149,18 +149,33 @@ private LuceneResult callLuceneDirectly(LuceneSearchView function, String queryS
         if (resultLimit > 0) {
             queryURI.param("limit", resultLimit);
         }
+        if (dbNameForLuceneSearch.equals(DatabaseSettings.COUCH_DB_USERS)) {
```
[the remark this answers is not in the document]
Comment (JaideepPalit) [reply libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/couchdb/lucene/LuceneAwareDatabaseConnector.java]: Changes done.

File: libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/couchdb/lucene/LuceneAwareDatabaseConnector.java [hunk 2]
```diff
@@ -149,18 +149,33 @@ private LuceneResult callLuceneDirectly(LuceneSearchView function, String queryS
...
-            connection = (HttpURLConnection) luceneResourceUrl.openConnection();
-            connection.setRequestMethod("GET");
-            connection.connect();
+            connection = makeLuceneRequest(luceneResourceUrl);
             int responseCode = connection.getResponseCode();
             if (responseCode == 200) {
-                ObjectMapper objectMapper = new EktorpLuceneObjectMapperFactory().createObjectMapper();
                 return objectMapper.readValue(connection.getInputStream(), LuceneResult.class);
             }
+            else {
```
[the remark this answers is not in the document]
Comment (JaideepPalit) [reply libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/couchdb/lucene/LuceneAwareDatabaseConnector.java]: Changes done.

**Review by smrutis1 — APPROVED** [review]
Code looks good to me. Tested successfully again after recent changes in staging instance.

**Pull request merged**